docs/core/get-started.md

```diff
@@ -47,7 +47,7 @@ You can get started developing .NET Core applications by following these step-by
 - Watch a video on [how to install and use Visual Studio 2017 and .NET Core](https://channel9.msdn.com/Blogs/dotnet/Get-Started-NET-Core-Visual-Studio-2017/).
 - [Getting started with .NET Core using the command-line.](tutorials/using-with-xplat-cli.md)
 
-See the [Prerequisites for Windows development](windows-prerequisites.md) article for a list of the supported Windows versions.
+See the [Prerequisites for Windows development](setup/index.md) article for a list of the supported Windows versions.
```

Comment:
All these links should be reviewed. They shouldn't blankly go to the index page. It depends on the context.

Here's to check the supported versions so it should go here: https://review.docs.microsoft.com/en-us/dotnet/core/setup/dependencies?branch=pr-en-us-15569&tabs=netcore30&pivots=os-windows

Reply:
yes, good catch. Back when this started that index page was the prereqs page.